en-US/snippets/2017/nov2017_mobile.lang

```diff
@@ -3,241 +3,253 @@
 ## NOTE: Brand names stay in English except when noted.
 
 
-# Mobile Banners
-# Text limit: 79 characters, including spaces.
+# Mobile banner
+## MAX_LENGTH: 79
```

Comment:
I've asked you to add the MAX_LENGTH directive, not to drop the comment ;-)

Localized files won't have the directive, and without the comment they won't have anything at all

Reply:
I have it at the beginning under note, like including spaces.  also, I will send out an email to stress on the character limitation. Do you think this will be enough? If not, I will add the note back. I feel there are more notes than there are strings in this file, lol!!

Reply:
No, they need an idea of how many characters they can use for eachi string, without waiting for an error.